src/main/src/main/java/org/geoserver/catalog/LayerGroupInfo.java

```diff
@@ -1,335 +1,362 @@
-/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
```

Comment:
The formatting of this file has been destroyed, is impossible to understand what changed :scream:, you may be able to fix this by running ``mvn fmt:format``.